app/src/common/utils/timeDateUtils.js

```diff
@@ -140,3 +140,5 @@ export const getMinutesFromTimestamp = (timestamp) => {
     .unix();
   return parseInt((moment(timestamp).unix() - currentUnix) / 60, 10);
 };
+
+export const formatValue = (value) => +value.toFixed(2);
```

Comment:
I think this name is too common. Also, does it relate to date and time?

Reply:
moved to numberUtils